function-server/spec/function-server/server-spec.js

```diff
@@ -74,18 +74,22 @@ describe("server tests", function() {
 
         it ("should return a FunctionError with fail function", async function() {
             const f = server.wrap(fail);
-            let r = await f({'context': null, 'payload': null});
-
-            expect(r.type).toEqual(server.FUNCTION_ERROR);
-            expect(r.message).toEqual("undefinedVariable is not defined");
+            try {
+                let r = await f({'context': null, 'payload': null});
+            } catch (err) {
+                expect(err.type).toEqual(server.FUNCTION_ERROR);
+                expect(err.message).toEqual("undefinedVariable is not defined");
+            }
```

Comment:
This works but would be better to use `expect().toThrow()` since this test will pass if there is no exception thrown.